crates/stackable-operator/CHANGELOG.md

```diff
@@ -8,7 +8,12 @@ All notable changes to this project will be documented in this file.
 
 - Rollout tracker for `StatefulSet` ([#833]).
 
+### Fixed
+
+- Invalid CRD schema for `StackableAffinity` contents. This was caused by the fields being optional and defaulting to `null`, while the custom schema marked the field as required ([836]).
```

Comment:
```suggestion
- Invalid CRD schema for `StackableAffinity` contents. This was caused by the fields being optional and defaulting to `null`, while the custom schema marked the field as required ([#836]).
```

Reply:
_Ignore this. I refreshed after you fixed it then lost access to the comment in the review UI_

Reply:
You are too fast xD :)